.pre-commit-config.yaml

```diff
@@ -45,14 +45,14 @@ repos:
           - "prettier"
           - "prettier-plugin-toml@0.3.1"
   - repo: "https://github.com/asottile/pyupgrade"
-    rev: e695ecd365119ab4e5463f6e49bea5f4b7ca786b
+    rev: v2.37.1
```

Comment:
It seems changing the the revision to a commit hash didn't prevent the version update.